source/glossary.rst

```diff
@@ -144,6 +144,14 @@ Glossary
      A system :term:`topology` that uses more than one server and more than one drive per server to host a MinIO instance.
      MinIO recommends Kubernetes for distributed deployments.
 
+   multipart upload
+     Multipart upload is an :s3-docs:`Amazon S3 function <mpuoverview.html>` that splits a single object into multiple parts for moving from one location to another.
+     The separate parts upload indepently from one another and then MinIO reconstructs them into the original object.
```

Comment:
I feel like we need an actor here:

Spitballing

```suggestion
The client uploads each part independently to MinIO, and MinIO manages reconstructing those received parts into the original object. 
```

Reply:
Will address in #595